starception/exception_handler.py

```diff
@@ -186,13 +189,30 @@ def highlight(value: str, filename: str) -> str:
         }
         lexer = mapping.get(extension)
         if lexer:
-            return highlight(value, lexer, HtmlFormatter(noclasses=True, nowrap=True, style=style))  # type: ignore
+            return highlight(value, lexer, HtmlFormatter(nowrap=True, style=style))  # type: ignore
         return value
     except ImportError:
         return value
 
 
+def get_pygments_styles() -> typing.Dict[str, str]:
+    try:
+        from pygments.formatters import HtmlFormatter
+
+        light_formatter = HtmlFormatter(style=THEME_LIGHT)
+        dark_formatter = HtmlFormatter(style=THEME_DARK)
+        return {
+            'code_theme_light': light_formatter.get_style_defs(),
+            'code_theme_dark': dark_formatter.get_style_defs(),
+        }
+    except ImportError:
+        return {'code_theme_light': '', 'code_theme_dark': ''}
+
+
 jinja = jinja2.Environment(loader=jinja2.PackageLoader(__name__))
+jinja.globals.update(
+    **get_pygments_styles(),
```

Comment:
probably i will copy these styles into a separate css file and remove these globals.